src/app/containers/Include/index.jsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/no-danger */
+import React from 'react';
+import { string } from 'prop-types';
+import { GridItemConstrainedMedium } from '#lib/styledGrid';
+import useToggle from '../Toggle/useToggle';
+
+const IncludeContainer = ({ html }) => {
```

Comment:
I'm happy to discuss this again with the team, I personally think since most of the logic here is really trivial for now it could be kept here. In the case that it becomes complex and needs to be maintained we could then consider moving it to psammead.

Reply:
Resolved: https://github.com/bbc/psammead/pull/3218#issuecomment-595710975